ctt-frontend/src/app/interfaces/user.ts

```diff
@@ -0,0 +1,3 @@
+export interface User {
+  userName: string;
```

Comment:
Is this just a wrapper, or is this also the base for the account component?

Reply:
This is planned to be our User object in the frontend and acts as a wrapper of `CognitoUserInterface`. To use it with the account service, it will have to be extendend with email etc.